schemas/sublime-color-scheme.json

```diff
@@ -16,58 +16,73 @@
     },
     "variables": {
       "type": "object",
-      "additionalProperties": {"type": "string"}
+      "additionalProperties": {
+        "type": "string",
```

Comment:
Is `"type": "string"` necessary? If not, remove it. 🙂

Reply:
@predragnikolic yeah afaik it's necessary because we also need values inside the `variables` object to color formatted strings